src/libraries/System.Net.Sockets/src/System/Net/Sockets/LingerOption.cs

```diff
@@ -41,5 +41,12 @@ public int LingerTime
                 _lingerTime = value;
             }
         }
+
+        public override bool Equals(object? comparand)
+        {
+            return comparand is LingerOption option && option.Enabled == _enabled && option.LingerTime == _lingerTime;
+        }
+
+        public override int GetHashCode() => base.GetHashCode();
```

Comment:
How would it break users? I assume this is what we get now e.g. the outcome would be identical. We _could_ create new `GetHashCode` but I doubt anybody is comparing the options for anything useful.

Reply:
It's very unlikely, but there is always a little chance that someone somewhere might be broken when a behavioral change happens, no matter how minor the change is. All I'm suggesting is to be explicit about it in the PR title, because the current title sounds like this is a test-only change, which is not true.